### ISSUE TITLE
Graphiql configuration not loaded in 1.0.0

### ISSUE THREAD

**Issue by l0co** [micronaut-projects/micronaut-graphql#6]
For such configuration:

```yaml
graphql:
  enabled: true
  graphiql:
    enabled: true
    path: /api/public/graphiql
    template-path: classpath:graphiql-index.html
```

`graphiql` configuration is not loaded [here](https://github.com/micronaut-projects/micronaut-graphql/blob/v1.0.0/graphql/src/main/java/io/micronaut/configuration/graphql/GraphiQLController.java#L73). The `graphiQLConfiguration` object still has defaults in its properties, instead of values loaded from config. The only reason the controller works is because of [this header](https://github.com/micronaut-projects/micronaut-graphql/blob/v1.0.0/graphql/src/main/java/io/micronaut/configuration/graphql/GraphiQLController.java#L52-L53), which uses appropriate properties from the config.

To make it working I need to repeat the same configuration this way:

```yaml
graphql:
  enabled: true
  graphiql: # thats the feed for @Controller annotation
    enabled: true
    path: /api/public/graphiql
    template-path: classpath:graphiql-index.html
  graphiQLConfiguration: # this is a workaround to load the proper config into graphiQLConfiguration object
    graphiql: 
      enabled: true
      path: /api/public/graphiql
      template-path: classpath:graphiql-index.html
```

It looks the [`GraphQLConfiguration`](https://github.com/micronaut-projects/micronaut-graphql/blob/v1.0.0/graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLConfiguration.java) is wrong.

**Comment by l0co** [comment]
Actually, it's enough to do this workaround and it starts to load config from the standard `graphiql` section:

```yaml
graphql:
  enabled: true
  graphiql: # thats the feed for @Controller annotation
    enabled: true
    path: /api/public/graphiql
    template-path: classpath:graphiql-index.html
  graphiQLConfiguration: # this is a workaround to load the proper config into graphiQLConfiguration object
```

**Comment by marceloverdijk** [comment]
@l0co thanks for reporting this. 
You are right the `GraphQLConfiguration` is wrong [here](https://github.com/micronaut-projects/micronaut-graphql/blob/v1.0.0/graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLConfiguration.java#L64)

I will fix it and make sure we have a test for this.

**Comment by marceloverdijk** [comment]
@l0co This is fixed now in `1.0.1` and will be synced to Maven Central shortly. Thx again for raising the issue.

**Comment by l0co** [comment]
I can confirm the fix works now in `1.0.1`.
